src/pages/heeyeolKim/Main/Main.js

```diff
@@ -0,0 +1,318 @@
+                                        <div>
+                                            <span className="authorMentionName">someone</span>
+                                            <span className="authorMentionComment">내 포즈가 어때?</span>
+                                        </div>
+                                    </div>
+                                    <div className="commentUnfold">
+                                        <a href="#!">댓글
+                                            <span>16개</span>
+                                            모두 보기
+                                        </a>
+                                    </div>
+                                    <div className="replierMentionContainer">
+                                        {this.state.comments.map((comment) => {
+                                            let {id, name, content} = comment;
+                                            // console.log(id)
+                                            return (
+                                            <CommentList
+                                                key = {id}
+                                                name = {name}
+                                                content = {content} />
+                                            )
+                                            // console.log(e.id);
+                                        }
+                                        )}
+                                    </div>
+                                </div>
+                                <div className="createdTime">
+                                    11시간 전
+                                </div>
+                            </section>
+                            <section className="createComment">
+                                <div className="createButton">
+                                    <div>
+                                        <button>
+                                            <div className="emoticonButton">
+                                                <i className="far fa-smile-wink"></i>
+                                            </div>
+                                        </button>
+                                        <input 
+                                        type="text" 
+                                        placeholder="댓글달기..."
+                                        value={this.state.newComment}
+                                        onChange = {this.createNewComment}
+                                        />
+                                    </div>
+                                    <div>
+                                        <button 
+                                        className="uploadComment"
+                                        onClick = {this.uploadNewComment}
+                                        >
+                                            <a href="#!">
```

Comment:
```suggestion
                                            <a href="#">
```
!가 있으면 뭔가 에러가 날 것 같은데 잘 모르겠습니다!

Reply:
오타를 쓰신 것 같아요

Reply:
수정 했습니다!